sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -348,12 +348,12 @@ instances:
         #
         # samples_per_hour_per_query: 4
 
-        ## @param lookback_window - integer - optional - default: 3600
+        ## @param lookback_window - integer - optional
         ## Queries whose last execution completed after the lookback window are excluded from
         ## metrics collection. Set a longer lookback window (in seconds) to capture infrequently
         ## run queries.
         #
-        # lookback_window: 3600
+        # lookback_window: <LOOKBACK_WINDOW>
```

Comment:
The effective default value (from the user's perspective) is `2*collection_interval`.

Reply:
`<LOOKBACK_WINDOW>` doesn't have meaning for the customer. wouldn't be better to put the actual value in there?

Reply:
CI runs `ddev validate config sqlserver -s` which is setting this value here. Do you know of a way to override it?

Reply:
no. but you could initialize it with 2*collection_interval and see if validate picks that up

Reply:
I see, I was able to do this by setting an `example` value. I chose 600 seconds.